examples/simple-asset-transfer/quorum/platform/docker-compose.yml

```diff
@@ -209,7 +209,7 @@ services:
       - PRIVATE_CONFIG=/qdata/tm/tm.ipc
       - NODE_ID=1
     networks:
-      quorum-examples-net:
+      quorum_quorum-examples-net:
```

Comment:
why change it here? After upgrading Quorum, one will need to re-do this change again. I would leave this config as close to original as possible. Instead you can update this network name in 'quorum/api/docker-compose.yaml`.

Reply:
✅ f5fb752887e03ba3b29c9364bc4dfad577f5f594